### DISCUSSION TITLE
Welcome to sd-1click-colab Discussions!

### DISCUSSION THREAD

**Discussion by nolanaatama** [nolanaatama/sd-1click-colab#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rlowe55** [comment]
as of today, getting this error when running gdrive_LoRA in google colab:
RuntimeError: Detected that PyTorch and torchvision were compiled with different CUDA versions. PyTorch has CUDA Version=11.7 and torchvision has CUDA Version=11.8

any help appreciated, thanks

**Reply by rlowe55** [reply]
It looks like the installed version on colab's VM image has been updated to "torch-2.0.0+cu118"

this line needs to be added to gdrive_LoRA.ipynb, or any other script being used

!pip install torch==1.13.1+cu117 torchvision==0.14.1+cu117 torchtext==0.14.1 torchaudio==0.13.1 torchdata==0.5.1 --extra-index-url https://download.pytorch.org/whl/cu117
